redfish/main.py

```diff
@@ -124,10 +124,9 @@
 standard_library.install_aliases()
 from builtins import object
 
-
-
 import json
-from urllib.parse import urlparse, urljoin
+import socket
```

Comment:
Does that require a change in the package deps ? If yes, could you create a tickt for that please ?